src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.R.Host.Client;
 
 namespace Microsoft.R.Host.Client.Session {
     public static class RSessionEvaluationCommands {
```

Comment:
Some of these may be able to be changed to use EvaluateAsync directly in the future, that will probably be a good time to do some reorganization.  I like the idea of separating them by area.